src/stories/Dialog/Dialog.stories.jsx

```diff
@@ -0,0 +1,25 @@
+import { fn } from '@storybook/test';
+
+import { Dialog } from '../../Components/Dialog/Dialog';
+
+export default {
+  title: 'Example/Dialog',
```

Comment:
For all `Dialog` stories could you please render a button, that displays the `Dialog` on clickand add `onClose` logic?
Otherwise there is no way to check documentation of the component, the Dialog renders on story open and covers everything:

![image](https://github.com/user-attachments/assets/0ef937cc-f4cc-480f-8bd9-020d8c04ab1c)